### PR TITLE
add pulkit0076

### LINKED ISSUE #368: Pulkit Singh
image

### PR DESCRIPTION
**Pull request by PulkitSinghDev**
resolve #368

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "name": "Pulkit Singh",
```
Comment (AllanRegush) [inline data.json:3]: Thank you for your PR 👍 Could you move your entry to the end?

**Review by AllanRegush — COMMENTED** [review]
Looking Good 👍

**Comment by AllanRegush** [comment]
@PulkitSinghDev Just checking in. Do you need any help? If you do let us know 🤓

**Comment by PulkitSinghDev** [comment]
> @PulkitSinghDev Just checking in. Do you need any help? If you do let us know 🤓
Yeah sure will let you know, if I want any help 😄

**Comment by AllanRegush** [comment]
Hi @PulkitSinghDev Just Checking in. Do you need any help? I'd love to have this PR Merged :nerd_face:

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]
Hey, @PulkitSinghDev 
Do you need any help/context on the suggestions?

**Comment by schmelto** [comment]
@PulkitSinghDev just checking in

**Comment by schmelto** [comment]
He, due to long inactivity I will close this PR now. Feel free to reopen it and make the necessary changes or create a new one. Thanks!

**Pull request closed without merging**